openfl-tutorials/interactive_api/Director_Pytorch_Kvasir_UNET/envoy_folder/start_envoy.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
 set -e
+ENVOY_NAME=$1
+SHARD_CONF=$2
+: "${ENVOY_NAME:=env_one}"
```

Comment:
Looks very tricky. I suggest to simplify it:
`ENVOY_NAME ="${ENVOY_NAME:-env_one}"`
or 
`ENVOY_NAME ="${1:-env_one}"`